surveys/2023-annual-survey/questions.md

```diff
@@ -594,6 +594,21 @@ Aspects:
 >
 > This question gathers data on the communities perceptions of certain aspects of Rust at this point in time.
 
+### How do you build your Rust projects?
+
+Type: select one (optional)
```

Comment:
I think that this might be a multiple select question, people can have multiple projects, use Cargo on some of them, use a different build system elsewhere.